SU-SOE/stanford_soe_helper
# SOE-2334 7.x 2.x responsive tweaks

# NOT READY FOR REVIEW

# Summary
- 

# Needed By (Date)
- Sprint end

# Criticality
- Fixed on prod, need to put into code.

# Steps to Test
- Switch to this branch
- May need to revert feature
- Navigate to __Path__
- Verify you see __Text__

# Affects 
- SoE (School of Engineering website)

# Associated Issues and/or People
## Related JIRA ticket(s)

## Related PRs

## More Information

## Folks to notify


# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
